rollbar/__init__.py

```diff
@@ -231,6 +231,7 @@ def _get_pylons_request():
     'locals': {
         'enabled': True,
         'safe_repr': True,
+        'scrub_varargs': False,
```

Comment:
I would rename this to either "include_varargs" or "remove_varargs" since we are deleting varargs and not actually "scrubbing" them. i.e. we are not changing the value to something like '***'.